webui/react/src/pages/ExperimentDetails/TrialsComparisonModal.module.scss

```diff
@@ -1,63 +1,50 @@
 .base {
-  background-color: var(--theme-background);
-  border: 1px solid var(--theme-surface-border);
-  display: flex;
-  flex-direction: column;
   height: 100%;
   overflow: auto;
 
-  .row {
-    align-items: center;
-    border-bottom: 1px solid var(--theme-surface-border);
-    display: grid;
-    grid-auto-columns: minmax(160px, 1fr);
-    grid-auto-flow: column;
-    grid-template-columns: 170px repeat(auto-fill, minmax(160px, 1fr));
-    justify-content: space-between;
-    min-width: fit-content;
+  table {
+    border-collapse: collapse;
+    table-layout: fixed;
+    width: fit-content;
 
-    & > div:first-child {
-      background-color: var(--theme-stage);
+    tr {
+      border-bottom: 1px solid var(--theme-surface-border);
+
+      &:nth-child(even) {
+        background-color: var(--theme-surface-weak);
+      }
+      &:nth-child(odd) {
+        background-color: var(--theme-surface-strong);
+      }
     }
-  }
-  .row.sticky {
-    position: sticky;
-    top: 0;
-    z-index: 1;
-  }
-  .row.spanAll {
-    display: flex;
-    left: 0;
-    position: sticky;
-  }
-  .cell {
-    align-items: center;
-    display: flex;
-    overflow: hidden;
-    padding: 8px;
-    text-overflow: ellipsis;
-    white-space: nowrap;
+    th,
+    td {
+      padding: 8px;
+      width: 180px;
+    }
+    .tableSelectCell {
+      padding: unset;
 
-    & > * {
-      margin: 0;
-      max-width: 100%;
+      > .tableSelectContainer {
+        left: 0;
+        padding: 8px 16px;
+        position: sticky;
+        width: fit-content;
+      }
+    }
+    thead > tr:first-of-type {
+      background-color: var(--theme-background);
+      position: sticky;
+      top: 0;
+      z-index: 10;
+    }
+    th:first-of-type {
```

Comment:
fix: add `background-clip: padding-box;` so that the background color doesn't cover the border

Reply:
Added `background-clip: padding-box;` in `tr` since thats where border is applied

Reply:
It doesn't work when it's applied there though.

Reply:
Probably I miss the context. What behavior do you expect? 
We use solid border (not dashed) so i think background color doesn't cover the border anyways 
Also, border is applied in `tr`, and `th` doesn't have border property. Adding `background-clip: padding-box;` in `th:first-of-type` doesnt change anything 


https://github.com/determined-ai/determined/assets/109113616/f7a3915e-0d7d-48f3-94eb-b7d254309b27

Reply:
Without background-clip:
![Screenshot 2023-07-18 at 15-48-21 Uncategorized Experiments - Determined](https://github.com/determined-ai/determined/assets/15078396/630f2c56-98d6-4db1-aa28-c4c6f9ee0cd4)

With background-clip:
![Screenshot 2023-07-18 at 15-48-57 Uncategorized Experiments - Determined](https://github.com/determined-ai/determined/assets/15078396/541b67e8-259e-4cd8-9b66-d6499144d3a9)

It makes the borders extend to the row titles.

Reply:
Ah i see. Seems like it works fine without it on Chrome but not on FireFox. Will fix it.
Thanks for clarifying

Reply:
looks like its a Firefox bug https://bugzilla.mozilla.org/show_bug.cgi?id=688556